py-polars/polars/dataframe/frame.py

```diff
@@ -8180,22 +8283,64 @@ def sum(
         ╞═════╪═════╪══════╡
         │ 6   ┆ 21  ┆ null │
         └─────┴─────┴──────┘
-        >>> df.sum(axis=1)
-        shape: (3,)
-        Series: 'foo' [str]
-        [
-                "16a"
-                "27b"
-                "38c"
-        ]
-
         """
+        if axis is not None:
+            issue_deprecation_warning(
+                "The `axis` parameter for `DataFrame.sum` is deprecated."
+                " Use `DataFrame.min_horizontal()` to perform horizontal aggregation.",
```

Comment:
@stinodego Looks like this should say `sum_horizontal`.

Reply:
You're right - fix underway. Thanks!